agent/run/loop_test.go

```
@@ -0,0 +1,181 @@
+	var reports []poll.Report
+	polls := 0
+	srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		switch r.URL.Path {
+		case "/api/v1/fleet/agent/poll":
+			polls++
+			var in struct {
+				ETag string `json:"etag"`
+			}
+			json.NewDecoder(r.Body).Decode(&in)
+			if in.ETag == "e1" && polls > 1 {
+				w.WriteHeader(304)
```

golangci-lint (2.13.2)

[error] "304" can be replaced by http.StatusNotModified

(usestdlibvars)